src/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,40 @@
+import { PropTypes } from '@dhis2/prop-types'
+import { Menu, MenuItem } from '@dhis2/ui'
+import React from 'react'
+import { useHistory, useRouteMatch } from 'react-router-dom'
+import { sections } from '../../pages/sections.conf'
+import styles from './Sidebar.module.css'
+
+const SidebarItem = ({ label, path }) => {
+    const history = useHistory()
+    const isActive = !!useRouteMatch(path)
+    const navigateToPath = () => history.push(path)
+
+    return (
+        <MenuItem
+            className={styles.sidebarItem}
+            onClick={navigateToPath}
```

Comment:
I see a menu item can also be [used as a link](https://ui.dhis2.nu/demo/?path=/story/actions-menu-menu-item--link). Since you want to route with js, but ideally still use a link for accessiblity (and proper semantics) I was thinking what you could do here.

Asked it in #ui [here](https://dhis2.slack.com/archives/CBM8LNEQM/p1623761501124700). Concensus was that you could use a href prop on the menuitem, and preventDefault in the onClick handler. I haven't tested whether this'll render an `<a>` so make sure it's correct, but something along those lines should do it.

Reply:
Actually it seems the preventdefault isn't necessary as that's already called automatically.

Reply:
The internal `onClick` mechanism automatically calls `preventDefault`, so simply supplying both should work

Reply:
I third this comment

Reply:
Nice I didn't realise MenuItem could take a `href`. Unfortunately `preventDefault` is called even if the user ctrl-clicks so links are not opened in a new tab, but this [should be an easy fix](https://github.com/dhis2/app-hub/blob/d4621ca3ab58fe5f92f9be00bc17e3e3f3e732c5/client/src/components/Header/DropdownMenuItemLink/DropdownMenuItemLink.js#L6-L16).